### PR TITLE
Update format string

### PR DESCRIPTION
**Pull request by honekun**
* **What kind of change does this PR introduce?**
Update composer and PHP libraries
Require PHP >=7.2 ore >=8.0
* Update 'view_item' format string,  %S is not valid for sprintf.

### CODE REVIEW

**Review by mannuel — APPROVED** [review]

**Review by adamf321 — APPROVED** [review]

File: src/Cpt.php [hunk 1]
```diff
@@ -194,16 +196,18 @@ public function init() {
...
-			'labels' => $this->labels,
-			// Array of supported fields ( title, editor, thumbnail, etc ).
-			'supports' => $this->supports,
-			// We use the query var 'store' as opposed to the post type 'acf-store'.
-			'query_var' => strtolower( $this->singular ),
-			// Triggers the handling of re-writes for this post-type.
-			'rewrite' => $this->rewrite,
-		));
+		$this->set_args(
+			array(
```
Comment (mannuel) [inline src/Cpt.php]: @honekun can we use the array short syntax here?

File: src/Cpt.php [hunk 2]
```diff
@@ -356,9 +360,11 @@ public function set_supports( $support ) {
 	 */
 	private function update_arg( $name = '', $value = '' ) {
 		if ( ! empty( $name ) ) {
-			$this->set_args( array(
-				$name => $value,
-			));
+			$this->set_args(
+				array(
```
Comment (mannuel) [inline src/Cpt.php:364]: @honekun can we use the array short syntax here?

File: src/Cpt.php (hunk 1 above)
Comment (honekun) [reply src/Cpt.php]: Updated array syntax.
thanks @mannuel

File: src/Cpt.php (hunk 2 above)
Comment (honekun) [reply src/Cpt.php:364]: Updated array syntax.
thanks @mannuel

**Review by mannuel — APPROVED** [review]

**Pull request merged**